removed forward slashes

loekd opened the pull request:
Removed forward-slashes from resource names, using names with slashes does not result in a valid token. (Code samples are correct.)

azurecla commented:
Hi __@siriusk-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

loekd committed:
removed forward slashes 

Removed forward-slashes from resource names, using names with slashes does not result in a valid token. (Code samples are correct.)
